Sources/PackageSigning/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# This source file is part of the Swift open source project
+#
+# Copyright (c) 2023 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+add_library(PackageSigning STATIC
```

Comment:
cc @compnerd `PackageSigning` is a new module

Reply:
@compnerd Judging from https://github.com/apple/swift-installer-scripts/pull/76 and https://github.com/apple/swift-package-manager/pull/4182 I don't think we need to add `PackageSigning` to apple/swift-installer-scripts? But we need to add `SwiftCrypto` back (https://github.com/apple/swift-installer-scripts/pull/172)?

Reply:
Right, if it is statically linked, it doesn't need to be distributed.  SwiftCrypto is being statically linked currently, so that is already present.